src/Course/List.hs

```diff
@@ -281,11 +258,9 @@ find =
 --
 -- >>> lengthGT4 infinity
 -- True
-lengthGT4 ::
-  List a
-  -> Bool
-lengthGT4 =
-  error "todo: Course.List#lengthGT4"
+lengthGT4 :: List a -> Bool
+lengthGT4 = (>4) . length . take 5
```

Comment:
oh... very clever! ;-)

Reply:
hehehe

Reply:
or you could have done this (qing did something similar) if you wanted to be super efficient, as yours is.

```
lengthGT4 = (_:._:._:._:._) = True
lengthGT4 = False
```

Reply:
right! :+1: